### PR TITLE
Update from Weblate

### PR DESCRIPTION
**Pull request by ubports-weblate**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -22,7 +22,7 @@ msgstr ""
 
 #: ../src/qml/NewGroupPage.qml:250
 msgid "#channelName"
```
Comment (Flohack74) [inline po/it.po:24]: @lduboeuf can you please have a look at the source code? I doubt this word should be translated.

**Review by Flohack74 — CHANGES_REQUESTED** [review]
There are maybe some words translated that should not be translatable

File: po/it.po (hunk 1 above)
Comment (Flohack74) [reply po/it.po:24]: @lduboeuf ping ^^

Comment (lduboeuf) [reply po/it.po]: i doubt it needs to be translated too,but in code you can see the translation function i18n.tr ... does it need to be fixed ?

Comment (lduboeuf) [reply po/it.po:24]: Edit: it is a placeholder text but i think it should be renamed "Channel Name" instead of #channelName

**Review by Flohack74 — APPROVED** [review]

**Pull request merged**